ocs_ci/deployment/deployment.py

```diff
@@ -370,12 +386,42 @@ def subscribe_ocs(self):
         if subscription_plan_approval == "Manual":
             wait_for_install_plan_and_approve(self.namespace)
 
+    def get_arbiter_location(self):
+        """
+        Get arbiter mon location for storage cluster
+        """
+
+        nodes = ocp.OCP(kind="node").get().get("items", [])
+
+        worker_nodes_zones = {
+            node["metadata"]["labels"].get("failure-domain.beta.kubernetes.io/zone")
```

Comment:
could `failure-domain.beta.kubernetes.io/zone` be saved as a constant?

Reply:
Let me do that

Reply:
Done